frontend/packages/dev-console/src/components/import/SamplesCatalog.tsx

```diff
@@ -34,13 +35,18 @@ const SampleCatalog: React.FC<SampleCatalogProps> = ({ match }) => {
         ) : (
           <CreateProjectListPage title={t('devconsole~Samples')}>
             {(openProjectModal) => (
-              <Trans t={t} ns="devconsole">
-                Select a Project to view the list of samples or{' '}
-                <Button isInline variant="link" onClick={openProjectModal}>
-                  create a Project
-                </Button>
+              <>
+                {t('devconsole~Select a Project to view the list of samples')}{' '}
+                {canCreateProject && (
+                  <Trans t={t} ns="devconsole">
+                    or{' '}
+                    <Button isInline variant="link" onClick={openProjectModal}>
+                      create a Project
+                    </Button>
+                  </Trans>
```

Comment:
I believe you missed using your new component here, or is there a difference that I didn't see?